utils/hct/hctbuild.cmd

```diff
@@ -102,9 +102,9 @@ if "%1"=="-Release" (
   set BUILD_CONFIG=Release
   shift /1
 )
-if "%1"=="-vs2017" (
```

Comment:
It looks like with the removal of -vs2017, AppVeyor script that launches this will still pass this option, causing it to ignore all subsequent options, rather than doing something reasonable.  This is why AppVeyor build failed.  We really need to improve the option parsing here.  Can we keep the matching of this option (and do nothing), rather than deleting it, then we should improve the option parsing in general by putting it in a loop, and reporting+failing on unrecognized options.

Reply:
Should also update appveyor.yml to remove -vs2017 option, I suppose.